cnf-certification-test/lifecycle/suite.go

```diff
@@ -484,3 +490,51 @@ func testPodPersistentVolumeReclaimPolicy(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func testCPUIsolation(env *provider.TestEnvironment) {
+	ginkgo.By("Testing pods for CPU isolation requirements")
+
+	// Individual requirements we are looking for:
+	// - Resource Requests and Limits must be provided and identical. This determines whether or not we run the rest of the tests.
+	// - CPU Requests and Limits must be in the form of whole units
+	// - 'runtimeClassName' must be specified
+	// - Annotations must be provided disabling CPU and IRQ load-balancing
+
+	podsMissingIsolationRequirements := make(map[string]bool)
+
+	for _, put := range env.Pods {
+		it := isolation.NewCPUIsolationTester(put)
```

Comment:
I would rename it with something else (I confused it with ginkgo "It"). Does it mean isolation test ?

Reply:
Yeah I was just thinking I-solation T-ester 😄 I can change it up.